feature_server/commands.py

```diff
@@ -453,7 +480,8 @@ def teleport(connection, player1, player2 = None, silent = False):
         silent = silent or player.invisible
         message = '%s ' + ('silently ' if silent else '') + 'teleported to %s'
         message = message % (player.name, target.name)
-    player.set_location(target.get_location())
+    x, y, z = target.get_location()
+    player.set_location(((x-0.5), (y-0.5), (z+0.5)))
```

Comment:
This looks slightly worrying - what if the target player was standing right next to a wall and (x-0.5) gets the teleporting player stuck in the wall? Perhaps the change was due to a player somehow getting stuck in the other player or in the ground - I think we need to do some testing before merging this change.